packages/core/package.json

```diff
@@ -46,7 +46,6 @@
     "@pixi/runner": "file:../runner",
     "@pixi/settings": "file:../settings",
     "@pixi/ticker": "file:../ticker",
-    "@pixi/utils": "file:../utils",
-    "@types/offscreencanvas": "^2019.6.4"
```

Comment:
Why was this dependency removed?

Reply:
TypeScript 5 already has a built-in OffscreenCanvas interface, so I think this dependency is no longer needed? It is causing some type error when checking the workers.

Reply:
This should be another PR

Reply:
Okay